src/njs_builtin.c

```diff
@@ -129,6 +130,15 @@ njs_builtin_objects_create(njs_vm_t *vm)
     njs_lvlhsh_init(&shared->keywords_hash);
     njs_lvlhsh_init(&shared->values_hash);
 
+
+    if (njs_cnst_hash_predefined.descr.elts_count == 0) {
```

Comment:
We can hide the details of predefined hash from the usercode, by moving this check to `njs_predefined_hash_init()`.
This way we can simplify `njs_predefined.h`